applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py

```diff
@@ -49,6 +49,17 @@ def setUp(self):
 
             SelectAndVerifyLinearSolver(ProjectParameters, self.skipTest)
 
+            solver_type = ProjectParameters["solver_settings"]["linear_solver_settings"]["solver_type"].GetString()
```

Comment:
merge conflict?

Reply:
Nope, we said to follow that structure

Reply:
I am 99,9% sure this is a merge conflict, check a few lines above!

Reply:
okay

Reply:
OK, probably, I will check later